templates/shared/base.html

```diff
@@ -1,16 +1,69 @@
 <!DOCTYPE html>
+<!--  dnssɐʍ
```

Comment:
I kind of didn't want to put ASCII art on the content pages cause I felt it would be a bunch of wasted bytes.

Reply:
I don't think the extra bytes here are significant relative to the rest of some of the pages like the Timeline lmao. But yeah this was a joke I came up with late at night and I've gotten annoyed at it today while debugging via Developer Tools because its so large it looks like gibberish for half the elements/source tab. I'll remove it from the branch.